templates/components/templates/page-layout--right-sidebar/page-layout--right-sidebar.twig

```diff
@@ -0,0 +1,11 @@
+<div class="page-layout page-layout--right-sidebar">
+  <div class="column column--head">
+    {{ headContent }}
```

Comment:
Twig variable naming conventions use "_" separators and not camel case.

Reply:
Thanks for the heads_up!  Fixed.